README.md

```diff
@@ -97,6 +98,12 @@ eclair-node-<version>-<commit_id>/bin/eclair-node.sh
 
 You can then control your node via the [eclair-cli](https://github.com/ACINQ/eclair/wiki/Usage) or the [API](https://github.com/ACINQ/eclair/wiki/API).
 
+This repository also contains a sample GUI ([eclair-node-gui](https://github.com/ACINQ/eclair/tree/master/eclair-node-gui)).
+
+:warning: The sample GUI should only be used for prototyping and demo purposes: it is unsafe to use it on mainnet.
```

Comment:
```suggestion
:rotating_light: The sample GUI should only be used for prototyping and demo purposes: using it on mainnet is unsafe.
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1929/commits/7474d3ce32e45f162e7b82ada0ab05396158eb9e